db/python/layers/web.py

```diff
@@ -257,7 +306,9 @@ async def get_project_summary(
         )
         seqr_links = self.get_seqr_links_from_project(project)
 
-        sample_rows = list(await self.connection.fetch_all(sample_query, values))
+        sample_rows_all = list(await self.connection.fetch_all(sample_query, values))
```

Comment:
Can you add a comment here and say no projects even have close to 10,000 rows, so this shouldn't be a memory / resource problem. But could be optimised in the future by limiting a random number, and then just saying "Over 10k" if the results have that many?